builtin/plugins/dposv3/dpos_test.go

```diff
@@ -495,6 +495,140 @@ func TestDelegate(t *testing.T) {
 	require.NotNil(t, err)
 }
 
+func TestUnbondAll(t *testing.T) {
+	pctx := createCtx()
+
+	oraclePubKey, _ := hex.DecodeString(validatorPubKeyHex2)
+	oracleAddr := loom.Address{
+		Local: loom.LocalAddressFromPublicKey(oraclePubKey),
+	}
+
+	// Deploy the coin contract (DPOS Init() will attempt to resolve it)
+	coinContract := &coin.Coin{}
+	coinAddr := pctx.CreateContract(coin.Contract)
+	coinCtx := pctx.WithAddress(coinAddr)
+	coinContract.Init(contractpb.WrapPluginContext(coinCtx), &coin.InitRequest{
+		Accounts: []*coin.InitialAccount{
+			makeAccount(delegatorAddress1, 1000000000000000000),
+			makeAccount(delegatorAddress2, 2000000000000000000),
+			makeAccount(delegatorAddress3, 1000000000000000000),
+			makeAccount(addr1, 1000000000000000000),
+			makeAccount(addr2, 1000000000000000000),
+		},
+	})
+
+	dpos, err := deployDPOSContract(pctx, &Params{
+		ValidatorCount: 21,
+		OracleAddress:  oracleAddr.MarshalPB(),
+	})
+	require.Nil(t, err)
+
+	// transfer coins to reward fund
+	amount := big.NewInt(1)
+	amount.Mul(amount, big.NewInt(1e18))
```

Comment:
So the DPOS contract is only given 1 LOOM here, that's probably not enough to pay out all the rewards in this test.